src/diffusers/pipelines/pipeline_loading_utils.py

```diff
@@ -554,6 +554,11 @@ def _get_final_device_map(device_map, pipeline_class, passed_class_obj, init_dic
                 loaded_sub_model = passed_class_obj[name]
 
         else:
+            sub_model_dtype = (
+                torch_dtype.get(name, torch_dtype.get("_", torch.float32))
+                if isinstance(torch_dtype, dict)
+                else torch_dtype
+            )
```

Comment:
Could be `default`? Considering how it will work for integrations, instead of say `{'transformer': torch.bfloat16, 'text_encoder': torch.float16, 'text_encoder_2': torch.float16, 'text_encoder_3': torch.float16}` for SD3 and `{'transformer': torch.bfloat16, 'text_encoder': torch.float16, 'text_encoder_2': torch.float16}` for Flux. Not a big issue because components can be got from `cls._get_signature_types()`.

Reply:
Yeah no strong opinions.